src/serverdlg.cpp

```diff
@@ -534,6 +544,31 @@ void CServerDlg::UpdateGUIDependencies()
     // server list is enabled and not the default address is used
     edtCentralServerAddress->setEnabled (
         !bCurUseDefCentServAddr && bCurSerListEnabled );
+
+    QString strStatus;
```

Comment:
Open to views on these but there needs to be a reasonable gap between the checkbox label and the status label, so the text should not be too long.